src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { Icon } from "../Icon/Icon";
+
+export type BreadcrumbItems = {
+  name: string;
+  path: string;
+}[];
```

Comment:
It might be nice to add a prop that passes a callback function to `onClick`. I know in QMR we were using the `useNavigate` hook from `react-router-dom` for the majority navigating between pages. Probably not the end of the world if we don't expose this, but could be a nice option.

If you did want to allow a callback to be passed, but limit how it could be used, you might be able to even type check for `NavigateFunction`. But maybe we shouldn't. Just a thought.

Reply:
This sounds interesting but out of scope to me. Is there a requirement to add this kind of functionality? I don't see it in the QMR version of the breadcrumbs, but maybe I'm missing something.

Reply:
It might not be in the breadcrumbs specifically, but `useNavigate` is a commonly used piece of `react-router-dom`. Should just require us to handle an additional param.

Reply:
Would this be as easy as having an option param onClick?: () => {}?

Reply:
Yes, that's what I'm suggesting. Both `onClick` and `path` would be optional.

Reply:
This isn't a hill I want to die on, but I do think there is a potential value add here.

Reply:
I think it makes sense to me now, Gavin let us know if you wanna talk it over

Reply:
Ah, I think I understand now. Yeah, I can add that